pkg/internal/cyberark/dataupload/testdata/example-1/agent.yaml

```diff
@@ -0,0 +1,102 @@
+cluster_id: example-cluster-id
+organization_id: example-organization-id
+data-gatherers:
+# gather k8s apiserver version information
+- kind: k8s-discovery
+  name: ark/discovery
+- kind: k8s-dynamic
+  name: ark/secrets
```

Comment:
@wallrj-cyberark Since the name here is ark-specific, does that mean that we will fetch secrets twice (once for TLSPK and once for the Secret solution?)